packages/server/src/websockets/builder.ts

```diff
@@ -102,10 +101,9 @@ export default class BuilderSocket extends BaseSocket {
   }
 
   emitTableUpdate(ctx: any, table: Table, options?: EmitOptions) {
-    // This was added to make sure that sourceId is always present when
-    // sending this message to clients. Without this, tables without a
-    // sourceId (e.g. ta_users) won't get correctly updated client-side.
-    table = processTable(table)
+    if (table.sourceId == null || table.sourceId === "") {
```

Comment:
Are we excluding undefineds for a reason?

Reply:
This is a little bit on faith. I'm trusting that the comment that was here before is correct, and any missing `sourceId` is a problem. I'm not sure _why_ it's a problem, so I went wide on this conditional.